Exec/gravity_tests/hydrostatic_adjust/Castro_ext_src.H

```diff
@@ -0,0 +1,58 @@
+#ifndef CASTRO_EXT_SRC_H
+#define CASTRO_EXT_SRC_H
+
+#include <prob_parameters.H>
+
+using namespace amrex;
+
+const Real H_0 = heating_peak;
+const Real W_0 = heating_sigma;
+const Real r_0 = heating_rad;
+const Real t_stop = heating_time;
+
+AMREX_GPU_DEVICE AMREX_FORCE_INLINE
+void do_ext_src(int i, int j, int k,
+                GeometryData const& geomdata, 
+                Array4<const Real> const& state,
+                Array4<Real> const& src,
+                GpuArray<Real, 3> center,
+                const Real dt)
+{
+    const Real* prob_lo = geomdata.ProbLo();
```

Comment:
this is missing the check on `time < t_step`.  For this problem, we only want to heat for some of the runtime.

Reply:
I think this means that we should always have `do_ext_src` pass in the time

Reply:
Yeah, I skipped this because I didn't want to have to pass another argument (especially as it's only required for this one problem), but will switch them all so time is passed in